JUnit version 4.11-SNAPSHOT-20120416-1530
.E
Time: 0.03
There was 1 failure:
1) life_the_universe_and_everything(HikerTest)
java.lang.AssertionError: expected:<42> but was:<54>
	at org.junit.Assert.fail(Assert.java:92)
	at org.junit.Assert.failNotEquals(Assert.java:646)
	at org.junit.Assert.assertEquals(Assert.java:127)
	at org.junit.Assert.assertEquals(Assert.java:471)
	at org.junit.Assert.assertEquals(Assert.java:455)
	at HikerTest.life_the_universe_and_everything(HikerTest.java:10)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:269)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:69)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:48)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:237)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:61)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:235)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:51)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:228)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:306)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:24)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:237)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:61)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:235)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:51)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:228)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:306)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:151)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:130)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:111)
	at org.junit.runner.JUnitCore.runMain(JUnitCore.java:92)
	at org.junit.runner.JUnitCore.runMainAndExit(JUnitCore.java:47)
	at org.junit.runner.JUnitCore.main(JUnitCore.java:38)

FAILURES!!!
Tests run: 1,  Failures: 1

